### ISSUE TITLE
Distinct SQL does not work properly

### ISSUE THREAD

**Issue by chanhengseang3** [introproventures/graphql-jpa-query#118]
I've tried with distinct parameter but i does not work correctly. I have added the record with same values in column and then use graphql query it  using `(distinct:true)`, but it still return duplicate result. I tried it in my code and in sample code of library too. I do't know if there are any other configue are needed or there are issues in syntax. 
![image](https://user-images.githubusercontent.com/26960131/56555522-6f2eae80-65bf-11e9-89eb-817a469603e4.png)

**Comment by igdianov** [comment]
@chanhengseang3 You can add the DISTINCT keyword to your query to tell Hibernate to return each Book entity only once. In JPQL the DISTINCT works on the entity level. It cannot be used to distinct scalar values from the query.

**Comment by molexx** [comment]
Sorry, I'm confused, I can't find the expected usage: distinct isn't in the schema for me?

![image](https://user-images.githubusercontent.com/210764/56657211-5c83aa80-668f-11e9-9ed1-37dcf7259f08.png)

**Comment by igdianov** [comment]
@molexx You need to enable distinct argument via GraphqlSchemaBuilder configuration : https://github.com/introproventures/graphql-jpa-query/blob/efc52785138b6e257cc03ca036fecbd92446efa0/graphql-jpa-query-schema/src/main/java/com/introproventures/graphql/jpa/query/schema/impl/GraphQLJpaSchemaBuilder.java#L205

**Comment by molexx** [comment]
Looks like it defaults to on, so should I expect this to only return the parent entity once?:

```
query {
  Humans (where: { appearsIn: {IN: [A_NEW_HOPE, EMPIRE_STRIKES_BACK]}}) {
    select {name}
  }
}
```

```
{
  "data": {
    "Humans": {
      "select": [
        {
          "name": "Luke Skywalker"
        },
        {
          "name": "Luke Skywalker"
        },
        {
          "name": "Darth Vader"
        },
        {
          "name": "Darth Vader"
        },
        {
          "name": "Han Solo"
        },
        {
          "name": "Han Solo"
        },
        {
          "name": "Leia Organa"
        },
        {
          "name": "Leia Organa"
        },
        {
          "name": "Wilhuff Tarkin"
        }
      ]
    }
  }
}
```

**Comment by igdianov** [comment]
@chanhengseang3 Thank for catching distinct gremlin. It is now fixed in 0.3.24

**Comment by chanhengseang3** [comment]
i don't think it work 
![image](https://user-images.githubusercontent.com/26960131/57120263-69dd1b00-6d9a-11e9-9470-2e1288e77db6.png)
